example/src/example_default.erl

```diff
@@ -1,38 +1,30 @@
 -module(example_default).
```

Comment:
This file was not only formatted but adapted to work with the `cowboy` version `trails` expects.